Src/GraphicLibs/Ncurses/NcursesLibrary.cpp

```diff
@@ -80,3 +80,30 @@ bool NcursesLibrary::drawEntity(IEntity &entity, std::pair<int, int> pos)
     reset_color_pairs();
     return true;
 }
+
+std::string NcursesLibrary::getLibraryName() const
+{
+    return "Ncurses";
+}
+
+Arcade::Evt NcursesLibrary::getInput() const
+{
+    Arcade::Evt evt;
+    int ch = getch();
+
+    switch (ch) {
+        case KEY_UP:
+            evt.key.key = Arcade::Evt::UP;
+            break;
+        case KEY_DOWN:
+            evt.key.key = Arcade::Evt::DOWN;
+            break;
+        case KEY_LEFT:
+            evt.key.key = Arcade::Evt::LEFT;
+            break;
+        case KEY_RIGHT:
+            evt.key.key = Arcade::Evt::RIGHT;
+            break;
+    }
```

Comment:
You are using mouse event instead of keyboard event, pls fix it as follow : 

KEY_UP = 'Z'   |  KEY_DOWN = 'S' ...
KEY_Z = 'z'      |  KEY_S = 's' ...etc